2026-02-18  Jakub Jelinek  <jakub@redhat.com>

	* go.test/go-test.exp (errchk): Use -translation binary instead of
	-encoding binary for TCL 9 compatibility.

--- gcc/testsuite/go.test/go-test.exp.jj	2026-01-02 09:56:10.367333137 +0100
+++ gcc/testsuite/go.test/go-test.exp	2026-02-18 08:55:48.239346228 +0100
@@ -82,9 +82,9 @@ proc errchk { test opts } {
 	set filename "errchk-$filename"
     }
     set fdin [open $test r]
-    fconfigure $fdin -encoding binary
+    fconfigure $fdin -translation binary
     set fdout [open $filename w]
-    fconfigure $fdout -encoding binary
+    fconfigure $fdout -translation binary
     while { [gets $fdin copy_line] >= 0 } {
 	if [string match "*////*" $copy_line] {
 	    puts $fdout $copy_line
